go/roothash/api/commitment/pool.go

```diff
@@ -445,6 +445,11 @@ func (p *Pool) TryFinalize(
 			// process the round, assuming that it is possible to do so.
 			if isTimeoutAuthoritative {
 				p.Discrepancy = true
+				// Arm the timer, but increase the roundTimeout as the backup workers should be
+				// given some more time to do the computation.
+				rearmTimer = true
+				// roundTimeout = 1.5*roundTimeout
+				roundTimeout = (15 * roundTimeout) / 10
```

Comment:
maybe move the `15` into a constant

Reply:
Good idea, added numerator/denominator constants.